docs/docs/Concepts/concepts-publish.md

```diff
@@ -12,6 +12,36 @@ Langflow provides several ways to publish and integrate your flows into external
 
 The **API access** pane presents code templates for integrating your flow into external applications.
 
+As of Langflow v1.5, all API requests require a `LANGFLOW_API_KEY`, even if `AUTO_LOGIN` is set to `True`.
+
+<details>
+<summary>Need help creating an API key?</summary>
+
+To generate a user-specific token to use with Langflow, do the following.
+
+1. Open the Langflow UI, click your user icon, and then select **Settings**.
+2. Click **Langflow API Keys**, and then click **Add New**.
+3. Name your key, and then click **Create API Key**.
+4. Copy the API key and store it in a secure location.
+5. Include your `LANGFLOW_API_KEY` in requests like this:
+    ```text
+    curl --request POST \
+     --url 'http://LANGFLOW_SERVER_ADDRESS/api/v1/run/FLOW_ID' \
+     --header 'Content-Type: application/json' \
+     --header 'x-api-key: LANGFLOW_API_KEY' \
+     --data '{
+       "output_type": "chat",
+       "input_type": "chat",
+       "input_value": "Hello"
+     }'
+    ```
+6. Alternatively, the API pane's code snippets include a script to detect your local `LANGFLOW_API_KEY`.
```

Comment:
```suggestion
6. Alternatively, the API pane's code snippets include a script to detect your local `LANGFLOW_API_KEY`.
```

It isn't clear whether this is a step telling the user to run the script or if it's just information that an active API key will be automatically used in those code snippets.

Reply:
Agreed, not a step